po/ko.po

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 #
 # Korean translation for the systemd.
-# Seong-ho Cho <shcho@gnome.org>, 2015.
+# Seong-ho Cho <shcho@gnome.org>, 2015, 2021.
 # Dongsu Park <dongsu@endocode.com>, 2015.
```

Comment:
I will add 2021 onto this line, later. :) Thank you, Dongsu Park!

Reply:
Thanks for pointing it out.
I have realized, my email address there also needs an update. Will do that later. :-)